package.json

```
@@ -0,0 +1,45 @@
+{
+  "name": "axe-linter-action",
+  "version": "1.0.0",
```

We should prevent a developer from accidentally publishing this:

```suggestion
  "version": "1.0.0",
  "private": true,
```